lib/currency_cloud/errors/error_utils.rb

```diff
@@ -10,5 +10,18 @@ def platform
                        end
       "#{base}#{implementation}"
     end
+
+    REDACTED_PARAMS = [:api_key, :login_id, :token].freeze
+
+    def redacted_params(params)
+      redacted = params.dup
```

Comment:
Could you please explain why you create a copy op params? That shouldn't be needed